src/ray/protobuf/experimental/autoscaler.proto

```diff
@@ -120,6 +118,14 @@ message NodeState {
 
   // The status of the node.
   NodeStatus status = 17;
+
+  message IdleInfo {
+    // Idle since a timestamp. If the node is currently not idle,
+    // this will be 0. An idle node is defined
+    int64 since_ts = 1;
```

Comment:
we can generalize this as "time_since_status_change"

Reply:
i see - then the IDLE status makes more sense.